docs/scheme_catalog_requirements.md

```diff
@@ -1,9 +1,9 @@
 
 # Scheme Catalog Requirements
 
-A Scheme Catalog Requirements Document specifies the metadata that a DCAT Catalog entry must contain for it to be Scheme-conforming. A conforming data source meets a common standard across a Scheme, where all Data Providers provide the same APIs, formats and meaning of data.
+A Scheme-conforming data source meets a common standard across a Scheme, where all Data Providers provide the same APIs, formats and meaning of data. A Scheme Catalog Requirements Document specifies the metadata that a DCAT Catalog entry must contain for it to be Scheme-conforming, and which roles may publish a conforming data source. 
```

Comment:
It's possible to interpret "all Data Providers provide the same APIs..." as meaning that all members of a scheme must implement all the scheme's APIs. This isn't the case. Could you make this clearer please?